File: templ.hs

-- Single variable
main = do
  x <- read <$> getLine
  putStrLn . show $ solve x

solve :: Int -> Int
solve = undefined

-- Single variable yes/no
main = do
  x <- read <$> getLine
  putStrLn $ if solve x then "YES" else "NO"

solve :: Int -> Int
solve = undefined


-- 2 variables
main = do
  (x:y:xs) <- map read . words <$> getLine
  putStrLn . show $ solve x y

solve :: Int -> Int -> Int
solve = undefined

-- Multi lines of lists
import Data.Traversable
import Data.List

main = do
  t <- read <$> getLine
  ps <- for [1..t] $ \x -> do
    read <$> getLine
  putStrLn . show $ solve ps

solve :: [Int] -> Int
solve = head

File: python_templates/templ.py

import numpy as np
words = lambda t : np.array(list(map(t, input().split())))

words = lambda t : list(map(t, input().split()))

def solve(x) :
  return x

x = int(input())
print(solve(x))

x = int(input())
if solve(x) :
  print("YES")
else:
  print("NO")

t = int(input())
li = list(map(int,input().split()))
print(solve(li))
